backend/graphql/core/Cargo.toml

```diff
@@ -24,4 +25,4 @@ serde = "1.0.126"
 serde_json = "1.0.66"
 thiserror = "1.0.30"
 tokio = { version = "1.29", features = ["macros" ] }
-async-std = "1.12"
+async-std = { version = "1", features = ["attributes", "tokio1"] }
```

Comment:
This cost me a tonne of time yesterday.

I think because of the need to call async functions in dgraph from with async graphql, this `tokio1` feature is needed but was hard to find!